students/tbrackney/Lesson6/mailroom4.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+"""
+File Name: mailroom4.py
+Author: Travis Brackney
+Class: Python 201 - Self paced online
+Date Created 5/8/2018
+Python Version: 3.6.4
+"""
+from operator import itemgetter
+
+
+def list_donors():
+    ''' Prints list of donors '''
+    for d in donors:
+        print(d)
+
+
+# Updated
+def add_donor(d_name):
+    '''Adds a new donor to donors'''
+    donors[d_name] = []
+
+
+def get_donation(d_name):
+    '''Appends a donation to existing donor '''
```

Comment:
note that triple double quotes are used for docs strings, not triple single quotes